sql/wsrep_sst.cc

```diff
@@ -515,16 +529,14 @@ static ssize_t sst_prepare_other(const char *method,
                  "wsrep_sst_%s "
                  WSREP_SST_OPT_ROLE " 'joiner' "
                  WSREP_SST_OPT_ADDR " '%s' "
-                 WSREP_SST_OPT_BASEDIR " '%s' "
                  WSREP_SST_OPT_DATA " '%s' "
-                 WSREP_SST_OPT_PLUGINDIR " '%s' "
                  WSREP_SST_OPT_CONF " '%s' "
                  WSREP_SST_OPT_CONF_SUFFIX " '%s' "
                  WSREP_SST_OPT_PARENT " '%d' "
                  WSREP_SST_OPT_VERSION " '%s' "
                  " %s '%s' ",
                  method, addr_in,
-                 mysql_home_ptr, mysql_real_data_home, opt_plugin_dir,
+                 mysql_real_data_home,
```

Comment:
I still feel that since all other params are passed through this route it is better to pass them here to maintain consistency.
(sst user name and password are sensitive information so can be passed through pipe).

Reply:
Moved back to command-line args